CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.2] - 2023-09-28 
+
+### Added
+* Clear `clearOnUnhover` property
+* Timestamp to `tapNodeData` and `tapEdgeData` properties
+* Timestamp to `mouseoverNodeData` and `mouseoverEdgeData` properties
+* Update `elements` positions when nodes are dragged
```

Comment:
```suggestion
## [UNRELEASED]

### Added
* [#195](https://github.com/plotly/dash-cytoscape/pull/195)
  - Add `clearOnUnhover` property.
  - Add timestamp to `tapNodeData`, `tapEdgeData`, `mouseoverNodeData`, and `mouseoverEdgeData` properties.

### Changed
* [#195](https://github.com/plotly/dash-cytoscape/pull/195) Update `elements` positions when nodes are dragged.
```

Standardizing how we handle changelogs.
- Add the version number and date only when creating a release, until then just call it `UNRELEASED`.
- Even though the headers are in past tense ("Added"), the convention is that individual items are written as full sentences in present tense.
- Each item should have a PR linked to it.

Also we didn't release v0.3.1 with the changes ascribed to it below, unless you think it's worth doing that (probably not, let's just keep going) please remove that section and fold those entries into `Added` and `Changed` (and adding `Fixed`) in this new `UNRELEASED` section, and format them according to these conventions (sorry I didn't notice that during the earlier PR review).